### PR TITLE
Add link to new ICS file

### PR DESCRIPTION
**Pull request by aj-stein-nist**
Add link to the new ICS file staged in scitt-community/scitt-web#23.

**NOTE:** Do not merge until the issue ☝️ is reviewed and merged. If not merged and deployed to website, do not approve this PR, do not merge.

### CODE REVIEW

**Comment by aj-stein-nist** [comment]
@SteveLasker with the related blocking PR merged can we do this one too.

File: profile/README.md [hunk 1]
```diff
@@ -10,8 +10,8 @@ To work as a community to facilitate adoption of IETF Supply Chain Integrity, Tr
 + Join the SCITT community mailing list at https://groups.io/g/scitt-community
 
 ## SCITT Community Meetings:
-+ SCITT Community Meetings: Bi-Weekly on Monday 08:00 PDT/11:00 EDT/15:00 UTC
-+ SCITT Community Bi-Weekly Meeting (iCal link): [Click Here](https://meet.google.com/rek-mbak-nxv)
++ SCITT Community Meetings: Bi-Weekly on Monday [08:00 PDT/11:00 EDT/15:00 UTC](https://time.is/compare/300PM_5_June_2024_in_UTC/Tokyo/California/New_York/London)
```
Comment (SteveLasker) [inline profile/README.md:13]: The time.is is a cool view. Could we add the iCal link here as well? Or, a link to the community page so we have one place to update if/when the meeting changes?

Comment (aj-stein-nist) [reply profile/README.md:13]: Argh, that's what I meant.  How did I screw that up? 😂

File: profile/README.md [hunk 2]
```diff
@@ -10,8 +10,8 @@ To work as a community to facilitate adoption of IETF Supply Chain Integrity, Tr
...
++ SCITT Community Bi-Weekly Meeting (iCalendar (ICS) link): [Click Here](https://meet.google.com/rek-mbak-nxv)
```
Comment (aj-stein-nist) [inline profile/README.md:14]:
```suggestion
+ SCITT Community Bi-Weekly Meeting (iCalendar (ICS) link): [Click Here](https://scitt.io/assets/schedule.ics)
```

**Review by SteveLasker — APPROVED** [review]
LGTM

**Pull request merged**